demo/headers.py

```diff
@@ -0,0 +1,192 @@
+from dataclasses import fields
+
+import mesop as me
+
+
+@me.page(
+  security_policy=me.SecurityPolicy(
+    allowed_iframe_parents=["https://google.github.io"]
+  ),
+  path="/headers",
+)
+def app():
+  is_mobile = me.viewport_size().width < 640
+
+  with me.box(style=me.Style(margin=me.Margin(bottom=15))):
+    # Two section basic header with fluid width.
+    # As an example, we don't use mobile view here since the header is short enough.
+    with header(max_width=None):
+      with header_section():
+        me.text(
+          "Mesop", type="headline-6", style=me.Style(margin=me.Margin(bottom=0))
```

Comment:
side-note: I've wondered whether we should just reset the default margins for headings to 0. I find it quite annoying to set it to margin 0, and it would improve cross-browser consistency.

Reply:
Yeah that's an interesting idea. On one hand, I definitely agree that the default headline margins are kind of absurdly large.

I think having some sensible margin bottom is kind of good, so it doesn't always need to be tweaked. Since 0 probably wouldn't be great in all cases.